unlock-app/src/components/creator/CreatorLocks.js

```diff
@@ -66,5 +70,25 @@ const LockMinorHeader = styled.div`
   color: var(--darkgrey);
 `
 
-const LockHeaderButton = styled.div`
+const LockHeaderButton = styled.div``
+
+const LockHeaderButtonContainer = styled.div`
+  max-height: 25px;
+  float: right;
```

Comment:
Meh. I think we should be very careful not to use `float` with `grid` layout. You can achieve something similar with `align-self: end` if the parent container is a grid.

But there again I think we actually do not need that element, you could just use `ActionButton` with `align-self: end` since `LockHeaderRow` is a grid.